package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "dependencies": {
     "serverless-python-requirements": "4.1.1",
-    "serverless-plugin-scripts": "^1.0.2"
+    "serverless-plugin-scripts": "^1.0.2",
```

Comment:
LGTM, assume there is rationale for ^1.0.2 patch window and also the 1.x release window on these dependencies...

Reply:
The rational is to limit the churn of benefiting from minor improvements without having to update this repo.
As per the npm versioning contract of:
```
Backward compatible bug fixes | Patch release | Increment the third digit | 1.0.1
Backward compatible new features | Minor release | Increment the middle digit and reset last digit to zero
```

Reply:
1.x and ^1.0.2 are equivalent
https://docs.npmjs.com/about-semantic-versioning

Reply:
personally i'm not a huge fan of unpinned dependencies, but i'll leave it up to you if you think it's worth it to have those minor improvements

Reply:
I generally fall on the pinning side myself.  It is guaranteed to produce the desired result on build.

Reply:
Good feedback regarding predictability, that made me think about security as well and the risk of auto-upgrading dependencies we do not control in the deployment pipeline. Even though I do trust Datadog to not abuse their position, the other plugin is from some random guy. 

With pinned dependencies, How do we manage the upgrade cycle? How would you leverage automation to limit that effort?